### PR TITLE
etcd clean up pt2

### PR DESCRIPTION
**Pull request by anujmaheshwari1**
etcd clean up pt 2, removing etcd constants, certificates etc

### CODE REVIEW

**Comment by adilliadil** [comment]
Do we need to remove the etcd stuff inside windows files or will that be handled separately?

File: pkg/agent/datamodel/types.go [hunk 1]
```diff
@@ -315,18 +313,6 @@ type CertificateProfile struct {
 	KubeConfigCertificate string `json:"kubeConfigCertificate,omitempty" conform:"redact"`
 	// KubeConfigPrivateKey is the client private key used for kubectl cli and signed by the CA
 	KubeConfigPrivateKey string `json:"kubeConfigPrivateKey,omitempty" conform:"redact"`
-	// EtcdServerCertificate is the server certificate for etcd, and signed by the CA
```
Comment (xuto2) [inline pkg/agent/datamodel/types.go:318]: also remove L309 APIServerPrivateKey and L305 CaPrivateKey

**Review by xuto2 — APPROVED** [review]

**Comment by xuto2** [comment]
> Do we need to remove the etcd stuff inside windows files or will that be handled separately?

I assume @mainred @AbelHu will handle that for windows.

**Comment by AbelHu** [comment]
Thanks for the reminder, @xuto2. AFAIK, etcd is only run in Linux. We can check it after this pr is merged into master.

**Review by adilliadil — APPROVED** [review]
reviewed last iteration LGTM

**Pull request merged**